Assignment/solution_student/index.html

```diff
@@ -15,17 +15,69 @@
     <!-- YOUR MARKUP GOES HERE! -->
     <div class="container">
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+      <div id="header">
+        <h1>Relaxr</h1>
+        <nav>
+          <ul>
+            <li><a href="#">About</a></li>
+            <li><a href="#">FAQ</a></li>
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Contact Us</a></li>
+            <li><a href="#">Blog</a></li>
+          </ul>
+        </nav>
+      </div>
 
-      <main>
-        <p>Our HTML starter template!</p>
-      </main>
+      <div id="main">
+        <article>
+          <header>
+            <h1>How I implemented Relaxr in 2 weeks and changed my life.</h1>
+          </header>
+            <img src="images/blog_photo1.jpg">
+             <p><span class="firstcharacter">L</span>orem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```

Comment:
Instead of wrapping each first character in a span with a class, which is actually done very well, you could make things more dynamic by using a pseudo-selector `:first-letter` (https://developer.mozilla.org/en-US/docs/Web/CSS/%3A%3Afirst-letter). This magical selector will select the first letter in a `p` tag, **so**, you could just do `article p:first-letter` to target the first letter in paragraphs in each article, _but_, to get even more dynamic, what if an article has more than one paragraph? You probably still only want the first paragraph to have the first letter drop-capped, so you can combine it with another pseudo-selector: :first-of-type, which selects only the first element of that type in that particular container. So you could end up with `article p:first-of-type:first-letter` which would select the first letter of the first paragraph in each `article` element, and your blog posts should be separated out into individual articles as I mentioned earlier.